geospaas/vocabularies/models.py

```diff
@@ -29,6 +29,13 @@ def __str__(self):
 
 
 
+    class Meta:
```

Comment:
Can you remove two empty lines before Meta here and below?

Reply:
done